### ISSUE TITLE
Image breaks with Alpine 3.8

### ISSUE THREAD

**Issue by alexhaydock** [dperson/samba#149]
Alpine 3.8 was released recently and it'd be good to have this image updated to use it as a base.

I notice that your container uses `FROM alpine` without specifying a version, which should pull Alpine with the `:latest` tag (Currently Alpine 3.8), but since your Docker Hub image hasn't been rebuilt since the release of Alpine 3.8, it's still based on Alpine 3.7 and thus still works.

After forking your Dockerfile and updating it to use `FROM alpine:3.8`, it fails to run.

The container builds properly but shows this error message when run:

```
exit_daemon: STATUS=daemon failed to start: Failed to create session, error code 1
```

Alpine 3.8 provides Samba `4.8.2-r1`, while Alpine 3.7 provides Samba `4.7.6-r0`.

I don't have any answers for the time being, but I can only suspect it's something that's down to the change in Samba package version.

Your container currently works so this isn't a bug report per se, but I thought you might appreciate the warning that the next time the container is rebuilt it'll probably start failing to run as above.

Temporary fix is to fix the Alpine version at 3.7 in the `FROM` line. I don't know about a "proper" fix, but there'll be plenty of time to work something out before 3.7 is end-of-life'd.

Thanks for the container!

**Comment by alexhaydock** [comment]
Actually, I just spotted that this would probably be fixed by pulling #147

**Comment by dperson** [comment]
It's merged now. Thanks.
